### PR TITLE
LL-2245 (ManagerV2): placeholder for supported tokens added to search

### PR DESCRIPTION
**Pull request by LFBarreto**
Manager search query now shows if the searched token is supported
and redirects to the accounts or installs the app if needed.

localhost_8080_webpack_index html

+ some polishes:
* scheduled queue bar now static in animation
* wording capitalization issues
* Bytesize component now takes in account both KB and MB size and does the correct rounding for better readability

### Type

Feature

### Context

Managerv2

LL-2245

### Parts of the app affected / Test plan

Managerv2

### CODE REVIEW

**Comment by dasilvarosa** [comment]
LGTM

File: src/renderer/screens/manager/AppsList/Placeholder.js [hunk 1]
```diff
@@ -1,23 +1,128 @@
...
+  installed: InstalledItem[],
+};
+
+const Placeholder = ({ query, addAccount, dispatch, installed }: Props) => {
+  const found = useMemo(
+    () =>
+      tokens.find(
+        token =>
+          token.name.toLocaleLowerCase().includes(query.toLowerCase()) ||
+          token.ticker.toLocaleLowerCase().includes(query.toLowerCase()),
```
Comment (gre) [inline src/renderer/screens/manager/AppsList/Placeholder.js]: why use of `toLocaleLowerCase` on one side?

File: static/i18n/en/app.json [hunk 2]
```diff
@@ -73,7 +73,9 @@
     }
   },
   "byteSize": {
-    "kbUnit": "{{size}}Kb"
+    "bytes": "{{size}}bytes",
+    "kbUnit": "{{size}}Kb",
+    "mbUnit": "{{size}}Mb"
```
Comment (gre) [inline static/i18n/en/app.json]: Aren't we suppose to have a space between size and unit?

Comment (dasilvarosa) [reply static/i18n/en/app.json]: Yes please :)

**Comment by gre** [comment]
I think the both raised issues are acceptable for this to get in. we can fork the discussion in JIRA tasks.

**Comment by nabil-brn** [comment]
Reported minor issues are fixed there: https://github.com/LedgerHQ/ledger-live-desktop/pull/2623

**Pull request merged**